addons/l10n_br_website_sale/views/templates.xml

```diff
@@ -33,6 +33,47 @@
                 </div>
             </t>
         </div>
+        <!-- o_city must remain in DOM, otherwise the standard website_sale js breaks. -->
+        <input id="o_city" position="attributes">
+            <attribute name="class" separator=" " add="o_standard_address"/>
+        </input>
+        <input id="o_city" position="after">
+            <select id="o_city_id" name="city_id" class="form-select o_extended_address">
+                <option value="">City...</option>
+                <t t-foreach="cities" t-as="c">
+                    <option t-att-value="c.id" t-att-selected="c.id == city.id" t-att-code="c.id" t-att-state-id="c.state_id.id" t-att-zip-ranges="c.l10n_br_zip_ranges">
+                        <t t-esc="c.name" />
+                    </option>
+                </t>
+            </select>
+        </input>
+        <!-- put base_address_extended fields separately to be more user-friendly -->
```

Comment:
It would be best to suggest UX improvement to the eCommerce team directly. That way, every customer would benefit from the more user-friendly interface - not only Brazilians - and it would remove complexity in your localization.
```suggestion
        <!-- put base_address_extended fields separately to be more user-friendly -->
```